src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/ImportPricesService.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.pecs.jpc.config.TimeSource
+import uk.gov.justice.digital.hmpps.pecs.jpc.domain.price.Supplier
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.spreadsheet.inbound.price.PriceImporter
+import uk.gov.justice.digital.hmpps.pecs.jpc.util.loggerFor
+import java.time.Duration
+
+/**
+ * Service for bulk importing supplier journey prices from AWS S3.
+ */
+private val logger = loggerFor<ImportPricesService>()
+
+@Service
+class ImportPricesService(
```

Comment:
This is pulled out of the now renamed ImportReportsService.  Feels a bit cleaner and one less dependency in the ImportReportsService.